docs/src/modules/concepts/pages/distributed-systems.adoc

```diff
@@ -1,4 +1,4 @@
-= A foundation of fundamental distributed systems principles and patterns
+= Computer Science
```

Comment:
Computer science sounds too generic. If covers processor architecture, I/O bus, networking protocol, programming language design and whatnot. 

Why not **Distributed systems principles**